### DISCUSSION TITLE
?accented=plain Psalms

### DISCUSSION THREAD

**Discussion by ajmalton** [DivinumOfficium/divinum-officium#2611] [Developers corner]
There are vestiges of an option _accented=plain_ which would render Latin without the liturgical diacritics (áéíóúë).

It was implemented using a whole separate copy of the Psalter, instead of just replacing the characters like the ij transform.

Most vernaculars ignore this, except German which clones the Latin, English which provides a version without *'s and with different punctuation, and Hungarian which amazingly has a whole (more colloquial) different translation (!) that still of course _has_ diacritics since they're obligatory in Hungarian.

However, I can't see that this feature is still supported.  And the actual alternate translations (Latin-Bea, Polski-Newer) are considered to be different versions, not options.

If we want an accented=plain it should just be done using tr or s/// , and it should apply to all the text, not just the Psalter.   

If we don't want this (currently obsolete and broken) feature, then its vestiges should be removed.  

Removal is my preference: elimination from the code, and the data files (*/psalms/*) would be moved to somewhere in obsolete/ .

**Comment by APMarcello3** [comment]
I have no objections to this proposal.

**Reply by fiapps** [reply]
This change would not impact Breviarium Meum, which does not offer access to this option.

The alternative Hungarian translation could be turned into a variant translation like the newer Polish translation. I think that just requires moving it to an appropriate directory, e.g., `Magyar-Newer` and adding the translation name to the list. The only difficult thing would be to figure out what it should be called.

**Reply by ajmalton** [reply]
Good. And good idea.

Maybe (since this only affects the Psalms) every language could offer optionally one alternate, chosen by an option "use alternate psalter".

That would remove the anomalous "language dash" fallback, and would eliminate the appearance that Polski-Newer or Latin-Bea are more different than they are... and support other languages' alternates without adding to the UI needlessly.

**Reply by rl1234567890** [reply]
> 
> That would remove the anomalous "language dash" fallback, and would eliminate the appearance that Polski-Newer or Latin-Bea are more different than they are... and support other languages' alternates without adding to the UI needlessly.

Please make sure that   having parallel Gallican and Bea psalter remains a possibility. Now, if you select Latin, only one column is left.

**Reply by ajmalton** [reply]
Aha, good catch.  That's what Discussions are for!

**Reply by fiapps** [reply]
> Maybe (since this only affects the Psalms) every language could offer optionally one alternate, chosen by an option "use alternate psalter".
> 
> That would remove the anomalous "language dash" fallback, and would eliminate the appearance that Polski-Newer or Latin-Bea are more different than they are... and support other languages' alternates without adding to the UI needlessly.

I don't think it's a good idea to remove functionality. Even though its currently only used for Psalms, the language dash fallback is a more general solution. I've been thinking of using it to offer a newer Italian translation of the readings of matins.
